Source/WebCore/platform/ScrollTypes.h

```diff
@@ -151,6 +151,13 @@ enum class ScrollElasticity : uint8_t {
     Allowed
 };
 
+// Determines if rubber-banding should be enabled if the content size is less than the scroll view size.
+enum class ScrollBounceBehavior : uint8_t {
```

Comment:
Let's call this `RubberBandingBehavior` so it matches all the variable names. It's OK to have a disconnect at the API boundary (better than the type not matching variable names everywhere).